scripts/partyUtils.js

```diff
@@ -0,0 +1,30 @@
+// Adding a class for animation
+function addAnimClass(element) {
+    const key = Math.floor(Math.random() * 3);
```

Comment:
Need to move in another function, which you can do not export but implement above 
Name like generateRandomKey